app/services/reminder_processor.py

```
@@ -98,3 +110,146 @@ def process_due_reminders():
+
+
+def process_due_calendar_alarms():
+    """Send due notifications for portable calendar event alarms."""
+    stats = {'checked': 0, 'sent': 0, 'failed': 0, 'skipped': 0, 'errors': []}
+    now = datetime.utcnow()
+
+    alarms = (
+        CalendarAlarm.query
+        .join(CalendarEvent)
+        .filter(
+            CalendarAlarm.is_enabled == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `CalendarAlarm.is_enabled:` for truth checks

Replace with `CalendarAlarm.is_enabled`

(E712)